packages/dds/cell/src/cell.ts

```diff
@@ -220,7 +220,26 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>>
     /**
      * Call back on disconnect
      */
-    protected onDisconnect() {}
+    protected onDisconnect() { }
+
+    /**
+     * Apply inner op
+     * @param content - ICellOperation content
+     */
+    private applyInnerOp(content: any) {
```

Comment:
This could be `ICellOperation` instead of `any` since this method is private and we've already asserted this type before we call this method in both places